frontend/actions/src/main/java/com/virtuslab/gitmachete/frontend/actions/toolbar/OpenMacheteFileAction.java

```diff
@@ -56,8 +56,6 @@ public void actionPerformed(AnActionEvent anActionEvent) {
             /* message */ getString("action.GitMachete.OpenMacheteFileAction.notification.title.cannot-open"))))
         .getOrNull();
 
-    getGraphTable(anActionEvent).queueRepositoryUpdateAndModelRefresh();
```

Comment:
Wow WTF was the purpose of this... can you dig out any clue from the original PR that added this line?

Reply:
I searched the git history and looked at https://github.com/VirtusLab/git-machete-intellij-plugin/pull/272 but I cannot see anything that could explain that